src/ui/settings/__tests__/index.js

```diff
@@ -0,0 +1,345 @@
+import sinon from 'sinon';
+import expect from 'expect.js';
+import init from '..';
+import defaultsProvider from '../defaults';
+
+describe('ui settings', function () {
```

Comment:
I think all the anonymous functions can be replaced with fat arrow functions in this file, to get more ES6ish.